### ISSUE TITLE
AdMob Ads Not Showing in Android

### ISSUE THREAD

**Issue by siddharth3322** [unity-plugins/Unity-Admob#93]
I have just integrated Admob plugin within my game project. I only want to display Admob fullscreen ads in Android build of game.

I have followed this document from first to last to display ads:
[https://developers.google.com/admob/unity/start](url)

I was getting this kind of messages in debug log:
![yman_admob_errors](https://user-images.githubusercontent.com/3082151/42123564-c46310fe-7c71-11e8-917d-de4db5eff21c.png)

Please give me some solution for this.

**Comment by unity-plugins** [comment]
Are you using the last version plugin?and have import all resource in the plugin?
I get error some times,but it show ads success

**Comment by siddharth3322** [comment]
Yes I have just downloaded plugin and using for showing advertisement.
Also plugin files imported completely within project that I have checked but my Admob account is complete new, I have published any game using this.
It may be possible some process is remaining to perform, so can you guide me for this? 

Apart from this, I was doing Admob advertisement setup from few years so I am not a new person in this.

**Comment by unity-plugins** [comment]
You can test with google test id 

```
#if UNITY_IOS
        		 appID="ca-app-pub-3940256099942544~1458002511";
				 bannerID="ca-app-pub-3940256099942544/2934735716";
				 interstitialID="ca-app-pub-3940256099942544/4411468910";
				 videoID="ca-app-pub-3940256099942544/1712485313";
				 nativeBannerID = "ca-app-pub-3940256099942544/3986624511";
        	#elif UNITY_ANDROID
        		 appID="ca-app-pub-3940256099942544~3347511713";
				 bannerID="ca-app-pub-3940256099942544/6300978111";
				 interstitialID="ca-app-pub-3940256099942544/1033173712";
				 videoID="ca-app-pub-3940256099942544/5224354917";
				 nativeBannerID = "ca-app-pub-3940256099942544/2247696110";
			#endif
```

**Comment by siddharth3322** [comment]
@unity-plugins After using above Ids test ads get displayed properly in game.
Here you have log details:
![yman_admob_log_details](https://user-images.githubusercontent.com/3082151/42153841-d3e8a1c4-7e01-11e8-8dcf-1cdfc161f209.png)

So now what is actual problem - can you able to help me further ?

**Comment by unity-plugins** [comment]
Ads display properly ,then you can ignore that warn

**Comment by siddharth3322** [comment]
But what about actual (real) ads ?? I am not getting in device...

what to do for that ?? 
after game release will it started showing automatically ??

**Comment by unity-plugins** [comment]
I am tring to test if there some lib not added

**Comment by siddharth3322** [comment]
Okay sir, i am waiting for your side reply....
At present using actual app admob ids, I can't able to get ads within device.

**Comment by siddharth3322** [comment]
Any solution for me ?
Still I was waiting for some solution....

**Comment by unity-plugins** [comment]
Updated ,have a try again

**Comment by siddharth3322** [comment]
@unity-plugins you mean new plugin version available for download that I require to implement and try ??

**Comment by unity-plugins** [comment]
Yes,a new version available for download,download and replace the old one

**Comment by siddharth3322** [comment]
Is this correct place for me download latest plugin ?
[https://github.com/unity-plugins/Unity-Admob/archive/master.zip](url)
Because this plugin AdMob ads showing related code is different compare to mine.

I was downloading Unity AdMob plugin from here:
[https://github.com/googleads/googleads-mobile-unity/releases/tag/v3.13.1](url)
So what to do now? Still ads not displayed...

**Comment by unity-plugins** [comment]
https://github.com/unity-plugins/Unity-Admob/archive/master.zip

**Comment by siddharth3322** [comment]
Yes but I was using Plugin that available for download through AdMob official website and both plugins structure is different.

**Comment by unity-plugins** [comment]
You need change to this plugin.
The official plugin and the code of this plugin are very different and difficult to use. I am not familiar with it.
